coda/coda_mdstore/tests/test_views.py

```diff
@@ -591,22 +591,35 @@ def test_raises_http404_file_handle_is_falsy(self, rf):
         with pytest.raises(http.Http404):
             views.bagURLList(request, bag.name)
 
-    @pytest.mark.xfail(reason='Refactor is required.')
-    def test_output_text(self):
-        assert 0
+    def test_response_content(self, rf):
+        """Test the output contains domain and parsed tag attributes from getFileHandle."""
+        self.getFileHandle.return_value.url = 'https://coda/testurl'
+        # Mock what gets read from the manifest file.
+        self.getFileHandle.return_value.readline.side_effect = [
+            '<html coda/bag/123/manifest-md5.txt', '<html coda/bag/123/bagit.txt', '']
```

Comment:
Any further thoughts on making the returned text look more like the text file as shown in my previous comment?

Reply:
I think I got confused, instead of using the urls in "manifest-md5.txt", I used the url which kind of looks like the url for 'manifest-md5.txt'.

Reply:
Also, I really would like to get rid of the `<html` bit, because it is a text file, so that is why I put in checksums like what would be in the manifest text file.